lib/tools/apk-utils.js

```diff
@@ -53,19 +53,25 @@ apkUtilsMethods.isAppInstalled = async function isAppInstalled (pkg) {
  *
  * @param {string} uri - The name of URI to start.
  * @param {string} pkg - The name of the package to start the URI with.
+ * @param {StartUriOptions} opts
```

Comment:
Please add typedef for StartUriOptions (see how it is done in the example like I posted before)

Reply:
Ohk. Got it.